### PR TITLE
Lecture 6

### PR DESCRIPTION
**Pull request by fayyozbek**
Fix comments, rename branch, delete unnecessary spaces

### CODE REVIEW

**Review by AlexanderZnak — CHANGES_REQUESTED** [review]
Unhandled exception. OpenQA.Selenium.DriverServiceNotFoundException: The geckodriver.exe file does not exist in the current directory or in a directory on the PATH environment variable. The driver can be downloaded at https://github.com/mozilla/g
eckodriver/releases.
   at OpenQA.Selenium.DriverService.FindDriverServiceExecutable(String executableName, Uri downloadUrl)
   at OpenQA.Selenium.Firefox.FirefoxDriverService.CreateDefaultService()
   at OpenQA.Selenium.Firefox.FirefoxDriver.CreateService(FirefoxOptions options)
   at OpenQA.Selenium.Firefox.FirefoxDriver..ctor(FirefoxOptions options)
   at Program.Main(String[] arg) in C:\Users\Aliaksandr.Znak\RiderProjects\iTechArt-Fayyozbek-Fozilov-\Homework_QA_Automation\Program.cs:line 6

I did not manage to run your homework

**Comment by fayyozbek** [comment]
You cannot run my homework, cause you don't have necessary driver for firefox. So I change the browser from firefox  to chrome.

File: Lecture_6/Lecture_6.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+class Program{
```
Comment (TanyaBorisenko) [inline Lecture_6/Lecture_6.cs:5]: space

File: Lecture_6/Lecture_6.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (TanyaBorisenko) [inline Lecture_6/Lecture_6.cs:1]: remove space

File: Lecture_6/Lecture_6.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+class Program{
+    [Test]
+    public void Test(){
```
Comment (TanyaBorisenko) [inline Lecture_6/Lecture_6.cs:7]: space

File: Lecture_6/Lecture_6.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+       //step 1
+       IWebDriver driver= new ChromeDriver();//opening Chrome browser
+       driver.Url="https://uzmarketing.com/";// move to website by link
+       //step 2
+       driver.FindElement(By.XPath("//strong[text()='Services']")).Click();// looking for strong which text equal to "Services" and click it
+       IWebElement element= driver.FindElement(By.XPath("//a[contains(@href, '/services/marketing-tools/')]"));// looking for hyperlink which href contains the link to marketing tools  
+       Console.WriteLine(element.Text); // show expeted result
+       //step 3
+       driver.FindElement(By.ClassName("fa-lightbulb-o")).Click();// find object by class name and click it
+       element= driver.FindElement(By.XPath("//h4[text()='Graphic Design']"));// looking for h4 which text equal to "Graphic Design"
```
Comment (TanyaBorisenko) [inline Lecture_6/Lecture_6.cs:17]: space

File: Lecture_6/Lecture_6.cs [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+using OpenQA.Selenium;
+class Program{
+    [Test]
+    public void Test(){
+       //step 1
+       IWebDriver driver= new ChromeDriver();//opening Chrome browser
+       driver.Url="https://uzmarketing.com/";// move to website by link
+       //step 2
+       driver.FindElement(By.XPath("//strong[text()='Services']")).Click();// looking for strong which text equal to "Services" and click it
+       IWebElement element= driver.FindElement(By.XPath("//a[contains(@href, '/services/marketing-tools/')]"));// looking for hyperlink which href contains the link to marketing tools
```
Comment (TanyaBorisenko) [inline Lecture_6/Lecture_6.cs:13]: space

File: Lecture_6/Lecture_6.cs [hunk 6]
```diff
@@ -0,0 +1,25 @@
+﻿
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+class Program{
+    [Test]
+    public void Test(){
+       //step 1
+       IWebDriver driver= new ChromeDriver();//opening Chrome browser
+       driver.Url="https://uzmarketing.com/";// move to website by link
```
Comment (TanyaBorisenko) [inline Lecture_6/Lecture_6.cs:10]: space

**Review by TanyaBorisenko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**